connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -233,8 +233,9 @@ public boolean createTopic(NewTopic topic) {
                     continue;
                 }
                 if (cause instanceof UnsupportedVersionException) {
-                    log.error("Unable to use Kafka admin client to create topic descriptions for '{}' using the brokers at {}", topicNameList, bootstrapServers);
-                    throw (UnsupportedVersionException) cause;
+                    log.debug("Unable to use Kafka admin client to create topic descriptions for '{}' using the brokers at {}," +
```

Comment:
We should update the javadoc so it reflects that we'd no longer throw `UnsupportedVersionException`.